### ISSUE TITLE
I got this error when I login to admin

### ISSUE THREAD

**Issue by peapple** [hungvt/MagentoVAdmin#7]
Fatal error: Class 'VES_VAdmin_Helper_Data' not found in /home/domarsolutions/public_html/app/Mage.php on line 546

it appears on the footer.

and when i want to access the configuration tab, this error shows with a black page.

**Comment by maginot** [comment]
Yes. I started having this error today after installing using Magento Connect on my magento version1.9.2.2 
Still debugging and trying to find a solution.

**Comment by maginot** [comment]
I think it's something related to Compilation option. If it's enabled then disable it.
